Makefile.am

```diff
@@ -183,6 +185,14 @@ release:
 	$(MAKE) po-pull
 	$(MAKE) dist
 
+container-release:
+	$(CONTAINER_ENGINE) run \
+	$(CONTAINER_TEST_ARGS) \
+	$(CI_TEST_ARGS) \
```

Comment:
We might want to eventually reconsider names of some of the variables that are used by all our containers.